typescript/cli/src/context/types.ts

```diff
@@ -48,7 +49,7 @@ export interface CommandContext
   chainMetadata: ChainMap<ChainMetadata>;
   multiProvider: MultiProvider;
   multiProtocolProvider: MultiProtocolProvider;
-  altVmProvider: AltVMProviderFactory;
+  altVmProvider: Map<string, AltVM.IProvider>;
```

Comment:
This should probably be of type `ProtocolMap` so that we are enforced by the compiler to actually check that we have a key defined on the mapping when we try to access it